tests/blockchain/config.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
 
 parallel = True
-job_timeout = 60
+job_timeout = 90
```

Comment:
this is a pretty significant increase. I would hope we would try to avoid tests taking this long. Do you have a sense of what's pushing the timings this high?